### ISSUE TITLE
Can you share the complete code?

### ISSUE THREAD

**Issue by rlcgetUp** [GeoZcx/A-deeply-supervised-image-fusion-network-for-change-detection-in-remote-sensing-images#24]

**Comment by GeoZcx** [comment]
Thanks for your interest. I have uploaded the training code including epoch, batch size, number of training steps, optimizer, learning rate, etc.

**Comment by rlcgetUp** [comment]
Yes,thank you very much!!!

**Comment by rlcgetUp** [comment]
OK，thanks.




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Chenxiao ***@***.***&gt;; 
发送时间: 2022年3月23日(星期三) 上午10:46
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [GeoZcx/A-deeply-supervised-image-fusion-network-for-change-detection-in-remote-sensing-images] Can you share the complete code? (Issue #24)





 
Thanks for your interest. I have uploaded the training code including epoch, batch size, number of training steps, optimizer, learning rate, etc.

**Comment by songyvying** [comment]
Have you successfully reappeared?I want to know how the predict script is implemented.

**Comment by rlcgetUp** [comment]
Sorry,My code is not work.It stills doesen't work.




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年4月29日(星期五) 下午4:57
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [GeoZcx/A-deeply-supervised-image-fusion-network-for-change-detection-in-remote-sensing-images] Can you share the complete code? (Issue #24)





 
Have you successfully reappeared?I want to know how the predict script is implemented.

**Comment by rlcgetUp** [comment]
Can you share the&nbsp; trianing.py and predict.py of troch version. I will appreciate you very much.&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "GeoZcx/A-deeply-supervised-image-fusion-network-for-change-detection-in-remote-sensing-images"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年3月23日(星期三) 上午10:46
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [GeoZcx/A-deeply-supervised-image-fusion-network-for-change-detection-in-remote-sensing-images] Can you share the complete code? (Issue #24)





 
Thanks for your interest. I have uploaded the training code including epoch, batch size, number of training steps, optimizer, learning rate, etc.
